cli/service.go

```diff
@@ -116,10 +117,14 @@ func (s *service) NamespaceCreate(data Parameters) (*models.Namespace, error) {
 		return nil, ErrDuplicateNamespace
 	}
 
+	if data.TenantID != "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" {
```

Comment:
This kind of conditional worries me. What does this special tenant id offer?

@gustavosbarreto @eduardoveiga

Reply:
Same to me, i think this is not really necessary.